user-manager/src/main/resources/db/migration/V1.0__CREATE_CUSTOMERS_TABLE.sql

```diff
@@ -2,9 +2,9 @@ CREATE TABLE IF NOT EXISTS customers
 (
     id         BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
     name       VARCHAR(32),
-    email      VARCHAR(32),
+    email      VARCHAR(32) UNIQUE,
     password   VARCHAR(32),
-    wallet_id  BIGINT,
+    wallet_id  BIGINT UNIQUE,
```

Comment:
if this code is on prod, you cannot just change this sql. You have to create a new migration. But right now it's ok if it is not in prod. In that case just be sure that other team members remove migrations from db table and run it again to have a correct one.